react-backend/functions/checkExists.js

```diff
@@ -1,7 +1,7 @@
 const checkToken = require('../database/queries/cb/pwd_token_check');
 
-module.exports = token => new Promise((resolve, reject) => {
-    checkToken(token, (error, result) => {
+module.exports = (client, token) => new Promise((resolve, reject) => {
```

Comment:
I noticed that we're using the word client for both dbConnections and postmark client. Can this be more specific?